### PR TITLE
WIP - Timezones and more

### PR DESCRIPTION
**Pull request by fiji-flo**

### CODE REVIEW

File: src/components/profile/view/ViewPersonalInfo.vue [hunk 1]
```diff
@@ -145,6 +145,25 @@ export default {
...
+      this.localtime = this.getLocaltime();
+    }, 1000);
+  },
+  beforeDestroy() {
+    window.clearInterval(this.interval);
+  },
+  methods: {
+    getLocaltime() {
+      const options = { timeZone: this.timezone.value };
+      return new Date().toLocaleTimeString(navigator.language, options);
```
Comment (fiji-flo) [inline src/components/profile/view/ViewPersonalInfo.vue]: Super fancy.
I love the web.

File: src/components/profile/edit/EditMutationWrapper.vue [hunk 2]
```diff
@@ -28,16 +28,21 @@
           <button type="submit" class="button">Save</button>
         </div>
       </form>
+      <LoadingSpinner v-if="loading"></LoadingSpinner>
```
[the remark this answers is not in the document]
Comment (fiji-flo) [reply src/components/profile/edit/EditMutationWrapper.vue]: the `v-else` broke other things. I will to this right any how. Also it looks quite weird if everything vanishes and there's only a loading spinner.

File: src/components/ui/Combobox.vue [hunk 3]
```diff
@@ -100,6 +101,9 @@ export default {
   margin: 0;
   background-color: var(--white);
   box-shadow: 0 0 0.25em 0 var(--gray-30);
+  max-height: 20em;
+  overflow: scroll;
+  z-index: 1;
```
[the remark this answers is not in the document]
Comment (fiji-flo) [reply src/components/ui/Combobox.vue]: Could I move this cleanup tasks over to you :)

File: src/components/profile/edit/EditMutationWrapper.vue (hunk 2 above)
[the remark this answers is not in the document]
Comment (hidde) [reply src/components/profile/edit/EditMutationWrapper.vue]: Another pattern we could consider is a spinner _inside_ the save button, turning it into a ‘Saving…’ state.

I guess this discussion is one we'll have at some point, it doesn't block merging this PR.

File: src/components/ui/Combobox.vue (hunk 3 above)
[the remark this answers is not in the document]
Comment (hidde) [reply src/components/ui/Combobox.vue]: Absolutely! Will do!

**Pull request merged**